### DISCUSSION TITLE
Ability to change functions color ( like macro )

### DISCUSSION THREAD

**Discussion by FAKlR** [TheHerobrine/DarkerNodes#107] [Ideas]
Would be really nice to setup color for function aswell , i use color on macro to make category more obvious on the graph and sadly for functions it can't be done .

Could it be possible ?

**Comment by hugoattal** [comment]
@FAKlR Sadly, if it's not already a function of the engine, this is unlikely to be possible 😕...

**Reply by FAKlR** [reply]
Too bad that would have been a killer features , i still dont get why it's still not a things in the engine .
Anyway thanks for your answer !
